libs/coin-evm/src/specs.ts

```diff
@@ -43,6 +43,7 @@ const minBalancePerCurrencyId: Partial<Record<CryptoCurrency["id"], number>> = {
   ethereum: 0.001,
   ethereum_goerli: 0.001,
   ethereum_classic: 0.05,
+  neon_evm: 0.001,
```

Comment:
Probably unecessary as the neon coin & their gas fees are very cheap.
This map is generally used only when the coin is expensive leading to "1" unit of it being too big of a limit to consider an account able to execute transactions.